src/tests/JIT/Regression/JitBlue/Runtime_34587/Runtime_34587.cs

```diff
@@ -664,7 +664,7 @@ static bool ValidateVector256()
         {
             bool succeeded = true;
 
-            if (Avx.IsSupported)
+            if (Avx2.IsSupported)
```

Comment:
@EgorBo, the `ValidateVector128` case needs to be updated above (from SSE to SSE2), to ensure that it will pass the outerloop ISA jobs

Reply:
@tannergooding oops :| thanks, forgot to kick in outerloop jobs

Reply:
I think there is a known failure there still that I have to fix, so probably good that you didn't...

I should hopefully get that resolved today (its likewise related to a mischeck when `SSE2` but not `SSE` is disabled)